mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -426,8 +427,10 @@ private SourceStorageBatchClient getSourceStorageBatchClient(OkapiConnectionPara
   private void postProcessMarcRecord(Record record, InitialRecord rawRecord) {
     String matchedId = getValue(record, TAG_999, 's');
     if (StringUtils.isNotBlank(matchedId)) {
```

Comment:
Can we merge the two if condition into one?

Reply:
Hi @Abdulkhakimov. Done, thank you!